hack/make/go.mk

```diff
@@ -21,12 +21,15 @@ go/format: go/fmt go/gci
 go/vet:
 	go vet -copylocks=false $(LINT_TARGET)
 
+go/wsl:
+	wsl -fix ./pkg/...
+
 ## Runs golangci-lint
 go/golangci:
 	golangci-lint run --build-tags "$(shell ./hack/build/create_go_build_tags.sh true)" --timeout 300s
 
 ## Runs all the linting tools
-go/lint: go/format go/vet go/golangci
+go/lint: prerequisites/go-linting go/format go/vet go/golangci
```

Comment:
Shouldn't `go/wsl` be added here, not the tool install step?

Reply:
well both actually, nice catch :D